core/server/data/schema.js

```diff
@@ -161,12 +162,23 @@ var db = {
             uuid: {type: 'string', maxlength: 36, nullable: false},
             name: {type: 'string', maxlength: 150, nullable: false, unique: true},
             slug: {type: 'string', maxlength: 150, nullable: false, unique: true},
-            secret: {type: 'string', maxlength: 150, nullable: false, unique: true},
+            secret: {type: 'string', maxlength: 150, nullable: false},
```

Comment:
You removed the unique here - are we sure that current setups don't have this as a constraint in the DB?